modules/ROOT/pages/manage-apis-with-service-mesh.adoc

```diff
@@ -30,8 +30,13 @@ image::select-policies-for-service-mesh.png[Applying Policies to APIs]
 . Select any of the following policies to secure your APIs: 
 * *Client ID Enforcement* 
 * *JWT Validation* 
-* *Rate Limit* 
-* *Rate Limit SLA-Based* 
+* *Rate Limiting* 
+* *Rate Limiting SLA-Based* 
+* *Basic Authentication: Simple*
+* *Basic Authentication: LDAP*
+* *OpenID Connect OAuth 2.0 Token Enforcement*
++
+NOTE: For the OpenID Connect OAuth 2.0 Token Enforcement policy, you must have configured OpenID as your client management provider. 
```

Comment:
When two nouns are used together as an adjective to modify a third noun in a noun string, they need a hyphen between them: client-management provider.

Reply:
Done! Thanks!